src/EventListener/InitializeSystemListener.php

```diff
@@ -151,6 +167,9 @@ private function boot($routeName, $basePath)
         // Log PHP errors
         $this->iniSet('error_log', $this->rootDir . '/system/logs/error.log');
 
+        // Support legacy Session access
+        $this->initializeLegacySessionAccess();
```

Comment:
Because I wanted this to be as early as possible. There's no specific reason - I would have added a comment if that was the case :)